### ISSUE TITLE
CMS throwing incompatible error with Laravel's  Str slug function

### ISSUE THREAD

**Issue by bruinetplan** [statamic/cms#6984]
### Bug description

Started getting this error when trying to run my Statamic instance locally.

```Declaration of Statamic\Support\Str::slug($string, $separator = '-', $language = 'en') must be compatible with Illuminate\Support\Str::slug($title, $separator = '-', $language = 'en', $dictionary = [...])```

### How to reproduce

Unsure how to reproduce, it just started happening.

Most recent version of Statami.

Laravel Version: 9.38.0

### Logs

```shell
https://flareapp.io/share/x7K9kpYm#F1
```

### Environment

```yaml
Environment
Application Name: Laravel
Laravel Version: 9.38.0
PHP Version: 8.1.11
Composer Version: 2.4.3
Environment: local
Debug Mode: ENABLED
URL: 127.0.0.1/
Maintenance Mode: OFF

Cache
Config: NOT CACHED
Events: NOT CACHED
Routes: NOT CACHED
Views: CACHED

Drivers
Broadcasting: log
Cache: redis
Database: pgsql
Logs: stderr
Mail: smtp
Queue: sync
Session: redis

Statamic
Addons: 3
Antlers: runtime
Version: 3.3.25 PRO

Statamic Addons
edalzell/forma: 1.2
mitydigital/sitemapamic: 2.3.3
silentz/mailchimp: 2.9.1
```

### Installation

Existing Laravel app

### Additional details

I am able to resolve it by editing the file:
`vendor/statamic/cms/src/Support/Str.php`

Simply adjusting the line like so:
`public static function slug($string, $separator = '-', $language = 'en', $dictionary = ['@' => 'at'])`

**Comment by bruinetplan** [comment]
Noticed that my statamic version was lagging behind a few minor versions. This was resolved in 3.50 I think, used my 1 liner fix above to run the updater in the control panel and that has it working.

**Comment by jasonvarga** [comment]
> This was resolved in 3.50

Yup. Fixed in 3.3.50 👍
